api_pos_integration.md

```diff
@@ -0,0 +1,277 @@
+# PoS Integration
+
+This documentation describes the snabble API endpoints related to the
+Point of Sale (PoS) integration.  These endpoints are available on the
+`api` subdomain. See [General API access](api_general.md) for general
+information about api access.
+
+## Operations
+
+* [Get checkout process](#get-checkout-process): `GET /{project}/pos/checkout/id/{id}`
+* [Update checkout process](#put-checkout-process): `PUT /{project}/pos/checkout/id/{id}`
+* [Partially update checkout process](#patch-checkout-process): `PATCH /{project}/pos/checkout/id/{id}`
+
+### Data Model
+
+* [Checkout Process](#checkout-process)
+
+## Workflow
+
+The integration of snabble into a Point of Sale software system (PoS)
+is implemented through a handover of an identifier from the snabble
+app to the PoS using a standard QR-Code. After the handover the PoS
+can fetch and process the checkout. Finally, it updates it to reflect
+the outcome of the process.
+
+The QR-Code presented by the user contains the identifier of the
+checkout process. It looks like this:
+```
+snabble:752dd716-ec0c-11e8-8528-68f7286a148f
+```
+
+The string after the colon is the identifier of the checkout process.
+
+To build the URL of the process the base URL of the used snabble
+environment (see [General API access:
+Environments](api_general.md#environments)), the identifier of the
+project and the identifier of the checkout is needed. This is the URL
+template:
+
+```
+<base URL>/<project id>/pos/checkout/id/<checkout id>
+```
+ie. `https://api.snabble.io/test-project-12345/pos/checkout/id/752dd716-ec0c-11e8-8528-68f7286a148f`.
+
+After the PoS has accessed the identifier of the checkout process, it
```

Comment:
"has accessed the checkout process" Es greift ja auf den process zu die ID ist ja nur Mittel zum Zweck